extensions/podman/packages/extension/package.json

```diff
@@ -149,6 +149,12 @@
           "scope": "ContainerProviderConnectionFactory",
           "description": "Image Path (Optional)"
         },
+        "podman.factory.machine.image-uri": {
+          "type": "string",
+          "scope": "ContainerProviderConnectionFactory",
+          "description": "Image URL or image reference (Optional)",
+          "placeholder": "Use 'quay.io/podman/machine-os:version' for image reference"
```

Comment:
it seems that for Windows it'll be `https://quay.io/repository/podman/machine-os-wsl` so I don't know if we need like two properties, one being activated on windows and the others on macOS

I don't know also if the placeholder link is working for Linux users

Reply:
I did not mean to point to specific image. I can change placepolder to `Use 'registry/org/image:version` for image reference` to avoid confusion.

The explanation how to use the URI field is not something we can fit in UI. We can consider add page in docs on podman-desktop.io, change description to markupDescription and have a link to documentation there.

Reply:
It seems there is only tag 5.3 for https://quay.io/repository/podman/machine-os-wsl?tab=tags, but current release is v5.2.3.

Reply:
Should we just let it to be provided by user and put `Use 'registry/org/image:version' for image reference` as placeholder for upcoming release?

Reply:
yes let's keep it very generic